### PR TITLE
Styling for React-Redux project

### PR DESCRIPTION
**Pull request by RafaelEchart**
## Milestone review:

- Title now can be long (40 characters max) with the help of line-break: auto.
- Application is fully responsive with a mobile-first approach to desktop ( max 1300px).
- Added title and border-top for the New Book Input.
- Added notifications when a new book is added successfully and a notification when the inputs are not correctly filled.
- Updated README file. 
- Better styling for mobile menu

### Desktop view, notification when a book is added: 
image

### Mobile view, notification when inputs are not correctly filled:
image

### INSTRUCTIONS: 
- npm install
- npm run start

### CODE REVIEW

File: src/components/NewBook/index.js [hunk 1]
```diff
@@ -9,42 +9,45 @@ import './newBook.css';
 
 const NewBook = () => {
```
Comment (D4nielJ) [inline src/components/NewBook/index.js:10]: - [ ] When I change tabs from books to categories to books, the component books rerenders again and fetches the books from the API again. This is not the intended behavior and styling should not modify it. Let's try to fix this. I will give you a tip: you need to check if you have fetch already the books when you `dispatch(loadBooks())` :wink: 

bookstore-rerender

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @RafaelEchart, 

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :fire: :rocket: :sparkles: 
- I'm amazed, congratulations, what it refers to the styling it looks GREAT!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/NewBook/index.js (hunk 1 above)
Comment (RafaelEchart) [reply src/components/NewBook/index.js:10]: Thanks! I didn't notice this, I just fixed this by using a simple if statement asking if the document is ready or not...

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @RafaelEchart, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**